### PR TITLE
Add service and file tests to the logging job

### PR DESCRIPTION
**Pull request by myadla**
OSPRH-9659

This covers adding the tasks from https://github.com/infrawatch/feature-verification-tests/pull/123 into a spearate PR, which will include:

    Update roles/common/tasks/main.yml
    Add roles/common/tasks/service_tests.yml
    Add roles/common/tasks/file_tests.yml
    Add service_tests and file_tests to the ci/logging* playbook
    Update the README file to document the configuration options

### CODE REVIEW

File: roles/common/README.md [hunk 1]
```diff
@@ -156,10 +171,42 @@ can be set at the play level.
           name: common
         vars:
           common_project_test_id: "RHOSO-12668"
+          common_service_nspace: openshift-logging
```
Comment (mgirgisf) [inline roles/common/README.md:174]: this need to be with the Verify Service example

**Review by mgirgisf — CHANGES_REQUESTED** [review]

File: roles/common/tasks/service_tests.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Verify Service - "{{ item }}"
+  ansible.builtin.shell:
+    cmd: |
+      oc get service -n "{{ common_service_nspace }}" "{{ item }}"
+  changed_when: false
+  register: output
+
+- name: Verify service is running "{{ common_service_test_id }}"
```
Comment (elfiesmelfie) [inline roles/common/tasks/service_tests.yml:9]:
```suggestion
- name: |
    Verify {{ item }} service is running
    {{ common_service_test_id }}
```

**Review by elfiesmelfie — CHANGES_REQUESTED** [review]

File: roles/common/README.md (hunk 1 above)
Comment (myadla) [reply roles/common/README.md]: Done

**Review by mgirgisf — APPROVED** [review]
Thanks Munesha

**Review by elfiesmelfie — APPROVED** [review]

**Comment by elfiesmelfie** [comment]
recheck

**Comment by myadla** [comment]
recheck

**Comment by elfiesmelfie** [comment]
recheck

**Pull request merged**